src/staticanalysis/bot/static_analysis_bot/clang/tidy.py

```diff
@@ -246,7 +247,7 @@ class ClangTidyIssue(Issue):
     '''
     ANALYZER = CLANG_TIDY
 
-    def __init__(self, revision, path, line, char, check, message, level='warning'):
+    def __init__(self, revision, path, line, char, check, message, level='warning', reliability='unknown'):
```

Comment:
It would be better to use an enum here:

```
class Reliability(enum.Enum):
  Unknown = 'unknown'
  High = 'high'
  Medium = 'medium'
```

and here you would have `..., reliability=Reliability.Unknown)`

This way you can assert you always have a `Reliability`